pennylane/ops/op_math/__init__.py

```diff
@@ -26,3 +26,5 @@
 from .control import ctrl, ControlledOperation
 
 from .pow_class import Pow
+
+from .symbolicop import SymbolicOp
```

Comment:
It will only be accessible via `qml.ops.op_math.SymoblicOp`, and potentially soon as `qml.ops.SymbolicOp`. I can further emphasize "DEVELOPER-ONLY" in the class docstring.

Reply:
Sounds great!